src/core/jit/model.rs

```diff
@@ -50,10 +52,39 @@ pub struct Field<Extensions> {
     pub name: String,
     pub ir: Option<IR>,
     pub type_of: crate::core::blueprint::Type,
+    pub include: Option<Include>,
```

Comment:
```suggestion
    pub ignore: Ignore,
```

```rust
enum Ignore {
    Never, // Default
    Always, // If a literal is used.    
    When(String)  // Represents when the variable name is true.
}
```

Reply:
won't work.. we need 2 information, i.e. if it is include or skip directive and the Value associated with it.

Reply:
```rust
enum Ignore {
    Never, // Default
    Always, // If a literal is used.    
    WhenTrue(Variable)  // Ignore when the variable is true.
    WhenFalse(Variable)  // Ignore when the variable is false.
}

struct Variable(String); 
```